### ISSUE TITLE
Apostrophe

### ISSUE THREAD

**Issue by xpersonas** [Atlantic18/DoctrineExtensions#1644]
From what I can tell, sluggable fields that have an apostrophe are replaced by the separator value. Rather than just removing. 

To me: /that-s-dave-s-cat
Would be better: /thats-daves-cat

But I don't see how I would go about setting this in the configuration. (correct me if I'm wrong) So what I'm wondering is, what's the best route to make this change? Is there something I can do with this extension still? Do I need to do something custom? 

 Can anyone help point me in the right direction here? I'm still working on my first symfony site.

**Comment by l3pp4rd** [comment]
you can [customize transliteration](https://github.com/Atlantic18/DoctrineExtensions/blob/master/lib/Gedmo/Sluggable/SluggableListener.php#L94) based on your wishes.

**Comment by nutii** [comment]
I have similar problem and I can't understand how to deal with.  I'm new with Symfony. Please, can anyone help me how to deal with problem?

**Comment by xpersonas** [comment]
Yeah, I ended up finding this on stack overflow...
http://stackoverflow.com/questions/11353685/symfony2-sluggable-multiple-separators

I'm somewhat new too. So I was a bit lost. I'm not sure if this is quite what l3pp4rd was suggesting, but I got it working this way. I was more or less trying to figure out how to override the "postProcessText" function.

So if this was not a good way to go about it, anyone can let me know. But this worked for me.
